File: CS/AppearancePrioritySample/Form1.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Windows.Forms;
using DevExpress.XtraGrid.Views.Base;
using DevExpress.XtraGrid;
using System.Drawing;

namespace GridControlDescendant {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
            myGridView1.ShowFindPanel();
            myGridControl1.DataSource = Data.CreateData(10);

            GridRowCellState state = GridRowCellState.Even | GridRowCellState.FocusedAndGridFocused;
            GridRowCellState newState = state & GridRowCellState.FocusedCell;
            newState = state & GridRowCellState.Even;

        }

        private void Form1_Load(object sender, EventArgs e)
        {
            StyleFormatCondition condition1 = new StyleFormatCondition();
            condition1.Appearance.BackColor = Color.Navy;
            condition1.Appearance.Options.UseBackColor = true;
            condition1.Condition = FormatConditionEnum.Expression;
            condition1.ApplyToRow = false;
            condition1.Expression = "[Value1] > 2";
            condition1.Column = gridColumn3;
            myGridView1.FormatConditions.Add(condition1);
        }
    }
}

File: VB/AppearancePrioritySample/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Windows.Forms
Imports DevExpress.XtraGrid.Views.Base
Imports DevExpress.XtraGrid
Imports System.Drawing

Namespace GridControlDescendant
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
			myGridView1.ShowFindPanel()
			myGridControl1.DataSource = Data.CreateData(10)

			Dim state As GridRowCellState = GridRowCellState.Even Or GridRowCellState.FocusedAndGridFocused
			Dim newState As GridRowCellState = state And GridRowCellState.FocusedCell
			newState = state And GridRowCellState.Even

		End Sub

		Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			Dim condition1 As New StyleFormatCondition()
			condition1.Appearance.BackColor = Color.Navy
			condition1.Appearance.Options.UseBackColor = True
			condition1.Condition = FormatConditionEnum.Expression
			condition1.ApplyToRow = False
			condition1.Expression = "[Value1] > 2"
			condition1.Column = gridColumn3
			myGridView1.FormatConditions.Add(condition1)
		End Sub
	End Class
End Namespace
